Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -677,7 +676,12 @@ def send_request(self, path: str, method: str = 'GET', body: dict | None = None,
         body = body if body is not None else {}
         params = params if params is not None else {}
 
-        if sc_api:
+        if custom_api:
+            server_url = demisto.params().get('url')
+            api = '/' + custom_api.strip(' /') + '/'
```

Comment:
What are you trying to do here why are you striping ' /' and then appending '/' from both sides?

Reply:
because the custom_api is a user defined value these stripping spaces and forward slashes and appending forward slashes is to make sure the user provided custom api has always a leading and trailing slash. 
All these examples:
```
' /api/custom'
'/api/custom '
' api/custom/'
'api/custom'
...
```
will lead to:
```
'/api/custom/'
```

Reply:
As discussed please remove the input manipulation. An exception can be returned in the case of invalid input.